submissions/drewzag/task_js_dom/styles.css

```diff
@@ -0,0 +1,205 @@
+*,
+*::after,
+*::before {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  height: 100vh;
+  background: linear-gradient(
+      145deg,
+      rgba(182, 181, 203, 1) 0%,
+      rgba(255, 0, 251, 1) 60%,
+      rgba(0, 51, 195, 1) 100%
+    )
+    no-repeat;
+  display: flex;
+  justify-content: center;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+a,
+a:link,
+a:visited,
+a:hover {
+  text-decoration: none;
+}
+
+.container {
+  width: 1000px;
+  height: 100%;
+  box-sizing: border-box;
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+}
+
+.menu__icon {
+  display: none;
+  height: 30px;
+}
+
+.nav {
+  border-radius: 30px;
+  margin: 20px;
+  overflow: scroll;
```

Comment:
Try to show the scrollbar to the user when needed. Showing them by default without scrolling looks not ok. [Watch the video](https://share.vidyard.com/watch/KX3KJGkAyxrzCPcRADm45b?)

Reply:
There are no scroll in my browser and I can't check it, but I change overflow to auto

Reply:
I suppose you have a MacOS they do not display scrollbars by default but most users will see them)
You hide them on the navigation page but not for the content block. [Video](https://share.vidyard.com/watch/s6Sm5Sd6ohHCafpEFZ7tx3?)